deploy/kubernetes/releases/csi-digitalocean-dev.yaml

```diff
@@ -168,6 +168,19 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
+        - name: csi-resizer
+          image: quay.io/k8scsi/csi-resizer:v0.3.0
+          args:
+            - "--v=5"
+            - "--csi-address=$(ADDRESS)"
+            - "--leader-election"
```

Comment:
I think we only specify a single csi-resizer instance, do we?

[The docs](https://github.com/kubernetes-csi/external-resizer#recommended-optional-arguments) say about this parameter (**emphasis** by me):

>`--leader-election:` Enables leader election. **This is mandatory when there are multiple replicas of the same external-resizer running for one CSI driver.** Only one of them may be active (=leader). A new leader will be re-elected when current leader dies or becomes unresponsive for ~15 seconds.

If our example doesn't specify multiple instances, then we shouldn't need this, right?